packages/core/src/store/level-db-store-wrapper.ts

```diff
@@ -0,0 +1,96 @@
+import levelTs from 'level-ts'
+import type Level from 'level-ts'
+import sublevel from 'sublevel'
+import { StoreWrapperInterface, StoreSearchParams } from './store-wrapper-interface.js'
+import path from 'path'
+import * as fs from 'fs'
+import { Networks } from '@ceramicnetwork/common'
+
+// When Node.js imports a CJS module from ESM, it considers whole contents of `module.exports` as ESM default export.
+// 'level-ts' is a CommomJS module, which exports Level constructor as `exports.default`.
+// This `default` name has no special meaning from ESM perspective. It is just yet another name.
+// Types provided by level-ts though make it appear as ESM default.
+//
+// So, here we untangle this mess, even if ugly.
+// We import type information separately from code information, and then make sure we can access
+// a properly typed constructor of `Level` (thus `LevelC`) exported from level-ts package.
+//
+// See also https://github.com/nodejs/node/blob/master/doc/api/esm.md#commonjs-namespaces,
+const LevelC = (levelTs as any).default as unknown as typeof Level
+
+export class LevelDbStoreWrapper implements StoreWrapperInterface {
```

Comment:
Also, names, maybe this: ?
- `StoreWrapperInterface` -> `IKVStore`,
- `StoreInterface` -> gone, because it is only used in `AbstractStore`,
- `AbstractStore` -> `ObjectStore`.

I also have a sort of phantom pain about using abstract classes in TS/JS. I recall a weird case, when a method implemented in a child class is not called from the parent class constructor. IMO, it might be slightly less error prone to pass `getKey`, `serialize`, and `deserialize` as parameters to the `AbstractStore` constructor. Then in a child class you'll have something like this:
```
constructor() {
super({
 getKey: (object: CommitID) => object.toString(),
 serialize: (value: AnchorRequestData) => JSON.stringify(value),
 deserialise: (serialized: any) => JSON.parse(serialized)
})
}
```

Reply:
Actually, as that would add some garbage to memory, IMO, the following more verbose version is slightly more appropriate, using `getKey` as an example. Imagine doing that to `serialise` and `deserialise`:

```ts
function getKey(object: CommitID) {
  return object.toString()
}

class AnchorRequestStore extends ObjectStore<CommitID, AnchorRequestData>  {
   constructor() {
    super(getKey, serialize, deserialise)
  }
}
```

Reply:
Side benefit: `AbstractStore` / `ObjectStore` is no longer abstract, and becomes properly testable in isolation.